src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -212,7 +252,12 @@ const CreateImageWizard = () => {
           <WizardStep
             name="Review"
             id="step-review"
-            footer={<CustomWizardFooter disableNext={true} />}
+            footer={
+              <ReviewWizardFooter
+                isLoading={isCreationLoading}
+                onSubmit={onSubmit}
```

Comment:
We could get rid of these two props if we just moved the `useMutation` hook into `<ReviewWizardFooter>` and also moved the `onSubmit` handler there as well. 

Generally I've tried to avoid passing down things like `isLoading` as props... since they are cached by RTKQ, it is cheap to just get them by calling the hook in the component where they are being used. The goal being to avoid having to search through the code to figure out exactly _what_ is loading. In this case if you are in the `<ReviewWizardFooter>` component you see that an `isLoading` prop gets passed in, but to figure out exactly _what_ is loading you have to go back to this component (which is already getting quite large and messy). If we just call the `useMutation` hook inside `<ReviewWizardFooter>` then we can more quickly understand where `isLoading` is coming from. Hope that makes sense...

Reply:
Moved to `<ReviewWizardFooter>`